.travis.yml

```diff
@@ -16,17 +16,15 @@ services:
   - memcached
   - mongodb
   - postgresql
+  - couchdb
```

Comment:
The issue with this approach is it reverts back to CouchDB 1.6, when the current version is now 3.0 ... at least it works though!

I'll see if I can resolve this in another way, but this is a good backup if not.